packages/mgt-chat/src/statefulClient/GraphNotificationClient.ts

```diff
@@ -78,6 +78,7 @@ export class GraphNotificationClient {
 
   // TODO: understand if this is needed under the native model
   private readonly onReconnect = (connectionId: string | undefined) => {
+    // eslint-disable-next-line @typescript-eslint/no-unsafe-call
```

Comment:
All of the no-unsafe-call suppressions here are invalid.

This is probably due to needing to rebuild and restart the eslint server after introducing the new files with an open editor